### PR TITLE
[automated-test/scd] Add documentation structure to SCD automated testing design

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
Currently, there are a few artifacts in the SCD automated testing design folder, but the context and purpose for them is not immediately obvious to a new reader.  This PR adds documentation to attempt to frame these artifacts in a useful way that will accommodate additional test design in the future.

### CODE REVIEW

File: interfaces/automated-testing/scd/design/u-space/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+The tests in this group validate requirements unique to U-space flight
+authorisations.
```
Comment (barroco) [inline interfaces/automated-testing/scd/design/u-space/README.md:2]: May I suggest to link to the [Article 10 of the regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e1041-161-1)?

**Review by barroco — APPROVED** [review]
Thank you for this, looks good to me. Just made a small suggestion

File: interfaces/automated-testing/scd/design/u-space/README.md (hunk 1 above)
Comment (BenjaminPelletier) [reply interfaces/automated-testing/scd/design/u-space/README.md]: Done.

**Comment by hrishiballal** [comment]
Thanks this looks great, no further comments from my side.

**Pull request merged**